UnitsNet.Tests/CustomCode/VolumePerLengthTests.cs

```diff
@@ -32,5 +32,9 @@ public class VolumePerLengthTests : VolumePerLengthTestsBase
         protected override double LitersPerMeterInOneCubicMeterPerMeter => 1000;
 
         protected override double OilBarrelsPerFootInOneCubicMeterPerMeter => 1.917134088;
+
+        protected override double CubicYardsPerFootInOneCubicMeterPerMeter => 0.3986633487670267521605;
```

Comment:
I was not able to verify these as I could not find any references, but I trust you checked it.

Reply:
A quick check would be
```
1 m² = 1.19599 yd²
1.19599 yd² * (1 yd / 3 ft) = 0.3986633 yd³ / ft
```